docs/source/_static/data_processing_examples/starmap.py

```diff
@@ -0,0 +1,192 @@
+"""
+STARmap processing example
+==========================
+
+This notebook demonstrates the processing of STARmap data using starfish. The
+data we present here is a subset of the data used in this
+`publication <starmap>`_ and was generously provided to us by the authors.
```

Comment:
this link probably renders correctly in sphinx gallery, but it does not render properly in the jupyter notebook. although i think having a sphinx gallery and a notebook is laudable, we have decided the priority for now is to make the notebooks render properly.

Reply:
same will be true for all other links in this notebook